service/src/globals/worker.rs

```diff
@@ -36,6 +35,12 @@ pub trait GetWorker {
 	fn get_worker<'a>(&self) -> RwLockReadGuard<'a, Option<Worker>>;
 }
 
+/// Trait for accessing a worker instance
+/// Prefer injecting this trait instead of using the associated functions of WorkerAccessorImpl
```

Comment:
```suggestion
/// Prefer injecting this trait instead of using the associated functions of WorkerAccessorImpl.
```

Reply:
Removed all together. Is mentioned in the `GlobalWorker` struct already